editor/vscode/browser-language-server/browserServerMain.ts

```diff
@@ -21,21 +21,25 @@ slint_init(slint_wasm_data).then((_) => {
         return true;
     }
 
+    async function load_file(path: string): Promise<string> {
```

Comment:
Note that this is currently type-checked, but for hygiene :)

```suggestion
    async function load_file(path: string): Promise<Uint8Array> {
```